modules/core/resources/META-INF/smithy/presence.smithy

```diff
@@ -0,0 +1,9 @@
+$version: "2"
+
+namespace alloy
+
+/// Use this trait to mark some field as nullable. This is to make
+/// a distinction between an optional feel that's missing and one
```

Comment:
an optional feel?

![image](https://user-images.githubusercontent.com/894884/208779078-e54880b4-767c-4867-af91-86f085ea1b86.png)

Reply:
> @daddykotex I'm not quite certain we can remove the `@default` trait instead of the `@defaultValue` trait, because of the following constraints : https://smithy.io/2.0/spec/type-refinement-traits.html?highlight=default#default-value-constraints
> 
> I think @lewisjkl and I had talked about something, in the context of smithy-translate, that would apply a best-effort transformation to use `@default` whenever possible, but `@defaultValue` would be kept to capture the default values that weren't complying with the constraints.

I was writing some sort of counter-argument but then it sort of _clicked_. I guess `defaultValue` were introduced to support `default` from open api and json schema at a time `smithy.api#default` was not available. But because `smithy.api#default` adds constraints on the default (like a string would need to match `length` requirement) this is not the case for open api or json schema. Naively converting to `smithy.api#default` would not work.

Reply:
did not send this reply to the right place lol